src/Okta.Idx.Sdk.E2ETests/Steps/Pages/EnrollPhoneAuthenticatorPageSteps.cs

```diff
@@ -33,6 +29,7 @@ public void WhenSheSelects(string p0)
         }
 
         [When(@"she inputs an invalid phone number")]
+        [When(@"She inputs a invalid phone number")]
```

Comment:
Should there be duplicate "When"s here?

Reply:
We have those two strings in *.feature files, but I agree, we don't need all of them. I will change the second condition to be exactly the same as the first one